travis/for-lint.sh

```diff
@@ -11,4 +11,6 @@
 mkdir -p app/code/Pulsestorm/Travis
 printf "<?php\nnamespace Pulsestorm\\Travis;\n\nclass Foo\n{\n}\n"  > app/code/Pulsestorm/Travis/Test.php
 
-# pestle.phar magento2:generate:crud-model Pulsestorm_Travis Thing
+## test generate:crud-model, rm InstallSchema backup file
+pestle.phar magento2:generate:crud-model Pulsestorm_Travis Thing
+rm app/code/Pulsestorm/Travis/Setup/InstallSchema.php.*.bak.php
```

Comment:
+1 for getting rid of the backup files -- what do you think about making this a bit more generic.  Something like ...

```
find app/code/Pulsestorm/Travis -type f -name '*.bak*' -exec rm '{}' \;
```

That'll get rid of any/all `.bak` files now and in the future.